components/arm/fake/fake.go

```diff
@@ -32,17 +42,41 @@ var fakeModelJSON []byte
 
 // AttrConfig is used for converting config attributes.
 type AttrConfig struct {
-	FailNew      bool   `json:"fail_new"`
-	FailValidate bool   `json:"fail_validate"`
-	ArmModel     string `json:"arm-model"`
+	ArmModel  string `json:"arm-model,omitempty"`
+	ModelPath string `json:"model-path,omitempty"`
```

Comment:
Can we call this ModelFilePath?

Reply:
This is what we are calling it on wrapper arm, if we update it here we should reflect it there as well.

Reply:
Agree the two should match.

IMO it being the path to a file is clearer

Reply:
will also update `wrapper.go` such that `AttrConfig.ArmName` becomes `AttrConfig.ArmModel`